tests/unit/models/test_evc_deploy.py

```diff
@@ -1228,6 +1278,91 @@ def test_deploy_direct_uni_flows(send_flow_mods_mock):
+        evc = TestEVC.create_evc_intra_switch()
+
+        evc.uni_a = get_uni_mocked(tag_value="untagged", is_valid=True)
+        evc.uni_z = get_uni_mocked(tag_value="untagged", is_valid=True)
+        expected_dpid = evc.uni_a.interface.switch.id
+        expected_flows = [
+            {
+                "match": {"in_port": 1, "dl_vlan": 0},
+                "cookie": evc.get_cookie(),
+                "actions": [
+                    {"action_type": "output", "port": 1},
+                ],
+                "priority": EVPL_SB_PRIORITY
+            },
+            {
+                "match": {"in_port": 1, "dl_vlan": 0},
+                "cookie": evc.get_cookie(),
+                "actions": [
+                    {"action_type": "output", "port": 1},
+                ],
+                "priority": EVPL_SB_PRIORITY
+            }
+        ]
+        evc._install_direct_uni_flows()
+        send_flow_mods_mock.assert_called_with(
+            expected_dpid, expected_flows
+        )
+
+        evc.uni_a = get_uni_mocked(tag_value="any", is_valid=True)
+        evc.uni_z = get_uni_mocked(tag_value="any", is_valid=True)
+        expected_dpid = evc.uni_a.interface.switch.id
+        expected_flows = [
+            {
+                "match": {"in_port": 1, "dl_vlan": "4096/4096"},
+                "cookie": evc.get_cookie(),
+                "actions": [
+                    {"action_type": "output", "port": 1},
+                ],
+                "priority": EVPL_SB_PRIORITY
+            },
+            {
+                "match": {"in_port": 1, "dl_vlan": "4096/4096"},
+                "cookie": evc.get_cookie(),
+                "actions": [
+                    {"action_type": "output", "port": 1},
+                ],
+                "priority": EVPL_SB_PRIORITY
+            }
+        ]
+        evc._install_direct_uni_flows()
+        send_flow_mods_mock.assert_called_with(
+            expected_dpid, expected_flows
+        )
+
+        evc.uni_a = get_uni_mocked(tag_value="any", is_valid=True)
+        evc.uni_z = get_uni_mocked(tag_value=100, is_valid=True)
+        expected_dpid = evc.uni_a.interface.switch.id
+        expected_flows = [
+            {
+                "match": {"in_port": 1, "dl_vlan": "4096/4096"},
+                "cookie": evc.get_cookie(),
+                "actions": [
+                    {"action_type": "set_vlan", "vlan_id": 100},
+                    {"action_type": "output", "port": 1},
+                ],
+                "priority": EVPL_SB_PRIORITY
+            },
+            {
+                "match": {"in_port": 1, "dl_vlan": 100},
+                "cookie": evc.get_cookie(),
+                "actions": [
+                    {"action_type": "output", "port": 1},
+                ],
+                "priority": EVPL_SB_PRIORITY
+            }
+        ]
+        evc._install_direct_uni_flows()
+        send_flow_mods_mock.assert_called_with(
+            expected_dpid, expected_flows
```

Comment:
Same here @italovalcy. I added `set_vlan` here since the output should match 100 (not sure). Should it not set the vlan value to the packet here?

Reply:
Handled on commit [6f82d45628cd580878c039930d836fbdbc1fd357](https://github.com/kytos-ng/mef_eline/pull/258/commits/6f82d45628cd580878c039930d836fbdbc1fd357)